examples/complete/providers.tf

```
@@ -35,7 +35,7 @@ terraform {
     }
     time = {
       source  = "hashicorp/time"
-      version = ">= 0.8.0"
+      version = "0.9.1"
```

Any particular reason for pinning the version?
```suggestion
      version = ">= 0.9.1"
```